app/controllers/state_file/questions/return_status_controller.rb

```diff
@@ -44,6 +48,55 @@ def refund_url
         end
       end
 
+      def tax_payment_url
```

Comment:
Sometimes we give different urls based on locale, worried about if we add spanish later if having this pattern will make it harder. Not sure if that will be the case for these links this time so this comment is non-blocking but just something to keep in mind

Reply:
Interesting point.  Do you have an example of handling this based on locale?